Dockerfile.local

```diff
@@ -1,15 +1,15 @@
-FROM node:8
+FROM node:12
 
 # Create our application directory
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-# Install Screwdriver API
-COPY package.json /usr/src/app/
+# Install Screwdriver Store
+COPY . /usr/src/app
 RUN npm install --production
 
-# Copy everything else
-COPY . /usr/src/app
+# Setup configuration folder
+RUN ln -s /usr/src/app/node_modules/screwdriver-store/config /config
```

Comment:
[ask] What is this symbolic link for?
The directory `/usr/src/app/node_modules/screwdriver-store/config` seems not exists.